src/page/template/content/preferences-account.htm

```diff
@@ -89,13 +89,11 @@
       <!-- ko if: isActivatedAccount() -->
         <label class="preferences-account-picture-button" for="self-upload-file-input" data-bind="attr: {title: t('tooltipPreferencesPicture')}">
           <participant-avatar params="participant: selfUser, size: z.components.ParticipantAvatar.SIZE.X_LARGE"></participant-avatar>
-          <!-- ko if: canEditProfile(selfUser()) -->
-            <input id="self-upload-file-input"
-                  type="file"
-                  data-bind="attr: {accept: z.viewModel.content.PreferencesAccountViewModel.CONFIG.PROFILE_IMAGE.FILE_TYPES.join(',')}, file_select: clickOnChangePicture"
-                  data-uie-name="do-select-picture"/>
-            <span class="icon-camera"></span>
-          <!-- /ko -->
+          <input id="self-upload-file-input"
+                type="file"
+                data-bind="attr: {accept: z.viewModel.content.PreferencesAccountViewModel.CONFIG.PROFILE_IMAGE.FILE_TYPES.join(',')}, file_select: clickOnChangePicture"
+                data-uie-name="do-select-picture"/>
```

Comment:
what's the rule for indentetion here?! two spaces, under id? random?

Reply:
Next line should start where `id` starts

Reply:
under the first attribute (id in this case)